go-minitwit/main.go

```diff
@@ -117,8 +103,8 @@ func bytesToString(bytes [16]byte) string {
 }
 
 func checkPasswordHash(userEnteredPwd string, dbpwd string) bool {
-	bytes := md5.Sum([]byte(userEnteredPwd))
-	str := bytesToString(bytes)
+	hash := md5.Sum([]byte(userEnteredPwd))
```

Comment:
can you explain why did you change it?

Reply:
Okay. Long explanation. Before our hashing of password was a bit incorrect in terms of the data type being passed to the db. 

So, in the new registerUser() (db-method), I convert the hashed password into a string to get the proper format for the db.

This gave another bug, because now the formats where messed up the checkPasswordHash() and i had to change that. But yeah, two lines so...

old one (buggy):
{32,44,185,98,172,89,7,91,150,75,7,21,45,35,75,112} ==  202cb962ac59075b964b07152d234b70

new one:

202cb962ac59075b964b07152d234b70  ==  202cb962ac59075b964b07152d234b70